tools/vagrant/simple/hello.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -e
+
+vagrant up
+
+vagrant ssh -c "sudo apt-get install -y git"
+
+vagrant ssh -c "git clone --depth=1 https://github.com/openwhisk/openwhisk.git"
+
+vagrant ssh -c "cd openwhisk/tools/ubuntu-setup && source all.sh"
+
+vagrant reload
+
+vagrant ssh -c "cd openwhisk && tools/db/couchdb/start-couchdb-box.sh whisk_admin some_passw0rd"
```

Comment:
Yes `start-couchdb-box.sh` needs to run before `ant clean` if not then `ant clean` fails because there is no couchdb-local.env file, so I need to run it to generate it
then `ant clean` will destroy the couchdb, so I need to recreate couchdb again.

This was the only to make it work.

@rabbah  But are you saying I can remove `ant clean` completely and just have `start-couchdb` once?